### DISCUSSION TITLE
How to disable validation warnings on console?

### DISCUSSION THREAD

**Discussion by guizoxxv** [element-plus/element-plus#11512] [Q&A] [answered]
When a validation error occurs, a console warning is displayed. How can I disable it?

The [async-validator](https://github.com/yiminghe/async-validator) library used in forms validation has a `suppressWarning` option, but I don't know how to use this option on `element-plus`.

Thank you.

**Comment by ryuhangyeong** [comment, marked as answer]
#11177 Are these answers helpful for you?

**Reply by guizoxxv** [reply]
> #11177 Are these answers helpful for you?

Thanks. Didn't saw this discussion. I understand that they don't occur in production, however it would be interestesting to allow disabling it in non-production mode. That is a functionality already present in the library.
